system-parachains/people/people-kusama/src/lib.rs

```diff
@@ -88,42 +98,6 @@ use xcm_runtime_apis::{
 	fees::Error as XcmPaymentApiError,
 };
 
-/// This determines the average expected block time that we are targeting.
-///
-/// Blocks will be produced at a minimum duration defined by `SLOT_DURATION`. `SLOT_DURATION` is
-/// picked up by `pallet_timestamp`, which is in turn picked up by `pallet_aura` to implement `fn
-/// slot_duration()`.
-///
-/// Change this to adjust the block time.
-pub const MILLISECS_PER_BLOCK: u64 = 6_000;
-pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
```

Comment:
People Kusama can not just switch, this would need a migration. Please leave at 6s.

Reply:
reverted slot duration to 6s for Kusama People Chain - https://github.com/polkadot-fellows/runtimes/pull/659/commits/d1056625018da9fa56c2133b15fedb35e9dac1ef